scripts/svg.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const xml = require('xml2js');
+const builder = new xml.Builder({
+    renderOpts: {
+        pretty: false
+    },
+    headless: true
+});
+
+const svgImport = /import \* as *(?<variable>.*) from '(?<path>.*\.svg)';/i;
+
+const svgfolder = path.resolve('./svg/');
+
+const filenames = fs.readdirSync(svgfolder);
+
+let svgStrings = {};
+filenames.forEach(filename => {
+    const buffer = fs.readFileSync(path.join(svgfolder, filename));
+    xml.parseString(buffer.toString('utf8'), (err, result) => {
```

Comment:
xml2js here use a callback, so you are not sure that it ran before continuing, I think you should use `xml2JS.parseStringPromise`.
You would be able to remove the builder defined l5 to 10 too.